app/data/models/payment-request.js

```diff
@@ -0,0 +1,38 @@
+module.exports = (sequelize, DataTypes) => {
+  const paymentRequest = sequelize.define('paymentRequest', {
+    paymentRequestId: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
```

Comment:
I think this needs the `schemeId` too from the Payment Request so we can isolate SFI 22 from Vets Visits etc.